include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -474,6 +587,9 @@ class argument_parser
      */
     void init(int const argc, char const * const * const argv)
     {
+        int argc_new{argc};
```

Comment:
![image](https://user-images.githubusercontent.com/2996863/58886274-21838680-86d3-11e9-94be-5fbb6b7f90dd.png)

Reply:
unfortunately, the version check option introduces an option that must be checked before the usual command line parsing (I need to handle the option before parse() is called as it might std::exit for format help) but it does not trigger a different format (like help, version, html export...) which means that the option must be removed from argv/argc before passing them to format_parse.
Any idea how to handle this differently?

Reply:
Copy in the parameter list?